content/hi/docs/reference/_index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: संदर्भ
+approvers:
+- chenopis
+linkTitle: "संदर्भ"
+main_menu: true
+weight: 70
+content_type: concept
+no_list: true
+---
+
+
+<!-- overview -->
+
+कुबेरनेट्स प्रलेखन के इस खंड में संदर्भ हैं।
+
+
+
+<!-- body -->
+
+## API संदर्भ
+
+* [शब्दकोष](/docs/reference/glossary/) - कुबेरनेट्स शब्दावली की एक व्यापक, मानकीकृत सूची।
+* [कुबेरनेट्स API संदर्भ](/docs/reference/kubernetes-api/)
+* [कुबेरनेट्स के लिए एक-पृष्ठ API संदर्भ {{< param "version" >}}](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/)
+* [कुबेरनेट्स API का उपयोग करना](/docs/reference/using-api/) - कुबेरनेट्स के लिए एपीआई का अवलोकन।
+* [API अभिगम नियंत्रण](/docs/reference/access-authn-authz/) - कुबेरनेट्स API एक्सेस को कैसे नियंत्रित करता है, इस पर विवरण।
+* [जाने-माने लेबल, एनोटेशन और दाग](/docs/reference/labels-annotations-taints/)
```

Comment:
not sure about this but should we use another word inplace of दाग as it has multiple meaning and I am not sure if that matches actual meaning of taints. Thoughts?

Reply:
I think we can use `टिप्पणी` instead `एनोटेशन`. Also I feel it it is better to transliterate `taints` (but I can suggest `अभिमिश्रण ` for the same).  Thoughts?

Reply:
I have replaced  एनोटेशन with टिप्पणी.